modules/aws/ec2.go

```diff
@@ -164,6 +164,35 @@ func DeleteAmiE(t *testing.T, region string, imageID string) error {
 	return err
 }
 
+func AddTagsToResource(t *testing.T, region string, resource string, tags map[string]string) {
+	err := AddTagsToResourceE(t, region, resource, tags)
+	if err != nil {
+		t.Fatal(err)
+	}
+}
+
+func AddTagsToResourceE(t *testing.T, region string, resource string, tags map[string]string) error {
```

Comment:
Could you add comments to these methods specifying what types of resources are supported and what the `resource` param should be?